docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+    You can specify a default value for ``o1`` in your project ``cumulusci.yml`` file and a default value for ``o2`` in your global ``cumulusci.yml`` file, and you'll see the expected result: both values are available in the project. (The default of ``o1`` is not exposed to other projects.)
+
+    If you change your project ``cumulusci.yml`` file to also specify a default value for ``o2``, this new default ``o2`` value takes precedence over the default ``o2`` value specified in your global ``cumulusci.yml`` file.
+
+
+Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``.../path/to/project/cumulusci.yml``
+
+**Windows:** ``...\path\to\project\cumulusci.yml``
+
+This ``cumulusci.yml`` file lives in the ``project`` directory and applies solely to this project. Changes here are committed back to a remote repository so other team members can benefit from the customizations. Configurations in this file apply solely to this project, and take precedence over any configurations specified in the global ``cumulusci.yml`` file, but are overridden by configurations in the local project ``cumulusci.yml`` file.
+
+
+Local Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``~/.cumulusci/project_name/cumulusci.yml``
+
+**Windows:** ``%homepath%\.cumulusci\project_name\cumulusci.yml``
+
+Configurations made to this ``cumulusci.yml`` file apply solely to the project with the given <project_name>, and take precedence over **all other** configuration scopes except the universal ``cumulusci.yml`` file. If you want to make customizations to a project, but don't need them to be available to other team members, make those customizations here.
+
+
+Global Configurations
+^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``~/.cumulusci/cumulusci.yml``
+
+**Windows:** ``%homepath%\.cumulusci\cumulusci.yml``
+
+Configuration of this file overrides behavior across **all** CumulusCI projects on your machine. Configurations in this file have a low precedence, and are overridden by **all other** configurations except for those that are in the universal ``cumulusci.yml`` file.
+
+
+Universal Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is one more configuration file that exists: the `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file that ships with CumulusCI itself. This file actually holds the lowest precedence of all, as all other scopes override what is contained in this file. That said, it contains all of the definitions for the standard tasks, flows, and org configurations that come standard with CumulusCI.
+
+The commands ``cci task info`` and ``cci flow info`` display all of the infomation regarding how a particular task or flow is configured. They display both the information present in the standard library alongside any customizations that have been added.
+
+
+
+Advanced Configurations
+-----------------------
+
+
+Use Tasks and Flows From a Different Project
```

Comment:
```suggestion
Use Tasks and Flows from a Different Project
```
"Prepositions that have fewer than five letters" at https://confluence.internal.salesforce.com/display/DOCTEAM/title+capitalization